src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/includeClassifier.js

```diff
@@ -31,7 +31,8 @@ const includeClassifier = ({ href, pathname }) => {
   // This determines if the type is supported and returns the include type name
   const includeType = supportedTypes[typeExtraction];
 
-  const pathnameIsAmp = pathname.endsWith('.amp');
+  const ampRegex = /(\.amp)/;
```

Comment:
I know this is literally one line but could we extract this out as utility and use the same utility for every amp url check in the project? At very least we could use the same utility for these two include components?

Also we should have a test to prove urls with `renderer_env` and without work I think?

Reply:
Thanks @andrewscfc - I tried to use the same utility for every amp url check but wasn't able to come up with a good approach (though maybe there is?) to refactor the function where it is used in [getRouteProps](https://github.com/bbc/simorgh/blob/latest/src/app/routes/utils/fetchPageData/utils/getRouteProps/index.js), and other other amp checks are passed down using React context. So I have just created a function for the two include components.

Reply:
Thanks for looking at this @Simon2828 . This amp check here could utilise the same regex: https://github.com/bbc/simorgh/blob/latest/src/app/routes/utils/fetchPageData/utils/getRouteProps/routeFallbackParams/index.js#L5..L7

In fact, I think you wouldn't need to use `url-parse` here anymore as this regex caters for `GET` parameters that are stripped by the library. My point was more to [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) up all the regex check implementations we have rather than any other ways we check Amp. I hope this helps?

Reply:
Thanks for explaining 😄